pkgs/development/compilers/go-jsonnet/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "go-jsonnet-${version}";
+  version = "0.12.1";
+  rev = "181c86d8157c7de54d052305b318be18b34d27d0";
+
+  goPackagePath = "github.com/google/go-jsonnet";
+
+  # regenerate deps.nix using following steps:
+  #
+  # go get -u github.com/google/go-jsonnet
+  # cd $GOPATH/src/github.com/google/go-jsonnet
+  # git checkout <version>
+  # dep init
+  # dep2nix
+  goDeps = ./deps.nix;
+
+  src = fetchFromGitHub {
+    inherit rev;
```

Comment:
Would be better to use a tag:
```rev = v${version};```

Reply:
I looked at that, but the current version tag is on the wrong commit on the upstream repo (I don't know why.)

Reply:
Hmm, actually, you are not fetching the release `0.12.1` but a non released version.
So, could you fetch a release by using the tag?

Reply:
Whoops, this is applicable now. Need to tweak once more.